android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java

```diff
@@ -251,6 +251,14 @@ public void trackSocialInteraction(String trackerId, String network, String acti
         }
     }
 
+	@ReactMethod
+	public void trackContentGroup(String trackerId, int contentGroupIndex, String category) {
+		Tracker tracker = getTracker(trackerId);
+		if (tracker != null) {
+			tracker.set(contentGroup(contentGroupIndex),category);	
```

Comment:
Have you tested this for Android? Where does `contentGroup` come from?

Also, could you fix whitespaces for this methods, `trackContentGroup` is indented further than the methods around.